### PR TITLE
🇨🇭 Add Switzerland

### PR DESCRIPTION
**Pull request by RVA2869**
I removed the Qids from the "unconfirmed" authorities/agencies (no official source) and I added the Swiss municipalities as well.

Fixes: #222

### CODE REVIEW

**Comment by Ainali** [comment]
Thanks for picking this up! The query works well, and this is a considerable addition, very cool!
You just forgot [to add](https://github.com/govdirectory/website/blob/main/ADD_A_COUNTRY.md#configuring-the-view) Switzerland to `views.yaml`.

**Comment by RVA2869** [comment]
Ignore adding and then removing... Busy day today.

File: queries/countries.rq [hunk 1]
```diff
@@ -42,6 +42,7 @@ WHERE {
     (wd:Q189 'Iceland' 'iceland' 'Current content includes ministries, courts and municipalities.' '')
     (wd:Q233 'Malta' 'malta' 'Current content includes ministries, regions and local councils.' '')
     (wd:Q1028 'Morocco' 'morocco' 'Current content includes ministries, embassies, regions, provinces and prefectures.' '')
+    (wd:Q39 'Switzerland' 'switzerland' 'Current content includes federal ministries, cantons and municipalities.')
```
Comment (Ainali) [inline queries/countries.rq:45]:
```suggestion
    (wd:Q39 'Switzerland' 'switzerland' 'Current content includes federal ministries, cantons and municipalities.' '')
```
Needed with the new federal subject possibility.

**Review by Ainali — APPROVED** [review]
Excellent addition, thanks for getting this ready!

This PR will also supersede #222 that can be closed after this has been merged.

**Pull request merged**